ChurchCRM/CRM
# Bugfix/3722 js api session expiration

#### What's this PR do?
Fixes the error that appears due to background javascript jobs when a session expires 

#### What Issues does it Close?

Closes #3722

#### Where should the reviewer start?

Review the changes that:
1) prevent these background AJAX tasks from running on pages (the password reset page)
2) observe the new AJAX error handler that will redirect a user to Login.php if the resultant JSON fails to parse, _and if_ the reason was Forbidden

#### How should this be manually tested?
Session timeout scenario:
1) Log into ChurchCRM.  
2) Either wait for your session to time out, or remove the session file from the server at ```sudo rm /var/lib/php/sessions/*```
3) You should not see any API call errors when the session times out, but rather you should be redirected to the login page.

forced password change scenario:
1) Log into ChurchCRM with a user account who has been flagged for "forced password change"
2) you should not see any API call errors, and you should not be redirected.

#### How should the automated tests treat this?
No.


#### Any background context you want to provide?

#### What are the relevant tickets?

#### Screenshots (if appropriate)

#### Questions:
- Is there a related website / article to substantiate / explain this change?
- Does the development wiki need an update?
- Does the user documentation wiki need an update?
- Does this add new dependencies?
